packages/client/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```diff
@@ -0,0 +1,58 @@
+%global debug_package %{nil}
+
+%global collection_namespace theforeman
+%global collection_name foreman
+%global collection_directory %{_datadir}/ansible/collections/ansible_collections/%{collection_namespace}/%{collection_name}
+
+%if 0%{?fedora} || 0%{?rhel} >= 8
+%global ansible_python python3
+%global pyyaml %{ansible_python}-pyyaml
+%else
+%global ansible_python python2
```

Comment:
With CentOS 7.7 out, and Python 3 available now, do we want to switch to python3 in all cases?

Reply:
No, unless ansible also switched to using python3.

Reply:
Well tell them to get with the times. It's been a day already!